.github/ISSUE_TEMPLATE.md

```diff
@@ -1,3 +1,13 @@
+# 🚨 Deprecation Notice 🚨
+
+Kubeaudit is planned for deprecation by October 2024.
+
+We are actively seeking maintainers who are interested in taking over the stewardship of this project. If you are passionate about continuing its development and maintenance, please reach out to us.
+
+For users looking for alternatives, we recommend transitioning to Kubebench, which offers similar functionality and is actively maintained.
+
+Thank you to the community for your contributions and support.
+
```

Comment:
q: should we do the same for PR templates? 🤔 https://github.com/Shopify/kubeaudit/blob/main/.github/PULL_REQUEST_TEMPLATE.md

Reply:
@dani-santos-code added this to PR templates.